ynput/ayon-launcher
# Fix Docker builds on Linux

## Changelog Description
Using Docker, AYON launcher can be built for multiple linux distributions like Centos 7, Rocky and Alma Linux 9, Ubuntu and Debian.

## Additional info
Running `./tools/docker_build.sh` will build using Docker ayon-launcher and installers and put them to `./build`. Those should be as *"static"* as possible.

## Testing notes:
1. Run `./tools/docker_build.sh rocky9`
2. Copy results to vanilla Rocky Linux 9 installation
3. All should work as expected
4. Test with `centos7` and `debian` too.
